products/jbrowse-web/src/jbrowseModel.js

```diff
@@ -39,6 +39,7 @@ export default function JBrowseWeb(
           defaultValue: false,
         },
         theme: { type: 'frozen', defaultValue: {} },
+        ...pluginManager.pluginConfigurationSchemas(),
```

Comment:
So this has the effect of having the top-level configuration have keys like:

```
MyPluginName: { bunch of config for MyPlugin }
```

right?

Reply:
yeah, seems like it, based on the tests